main.go

```diff
@@ -78,21 +80,66 @@ func main() {
 	} else if ctx.SetProtocol(gpgme.ProtocolAssuan) != nil {
 		log.Debugf("Cannot initialize Assuan IPC: %v. Disabling GPG and SSH watchers.", err)
 	} else {
-		gpgPubringPath := path.Join(gpgme.GetDirInfo("homedir"), "pubring.kbx")
-		if _, err := os.Stat(gpgPubringPath); err == nil {
-
-			requestGPGCheck := make(chan bool)
-			go detector.CheckGPGOnRequest(requestGPGCheck, notifiers, ctx)
-			go detector.WatchGPG(gpgPubringPath, requestGPGCheck)
-			go detector.WatchSSH(requestGPGCheck, exits)
-		} else {
-			log.Debugf("'%v' could not be found. Disabling GPG and SSH watchers.", gpgPubringPath)
+		var gpgPubringPath = path.Join(gpgme.GetDirInfo("homedir"), "private-keys-v1.d")
+		if _, err := os.Stat(gpgPubringPath); os.IsNotExist(err) {
+			fmt.Printf("Directory '%s' does not exist (you have no private keys).\n", gpgPubringPath)
+			return
 		}
+		var searchTerm = "shadowed-private-key"
+		var filesToWatch []string
```

Comment:
In Go in most cases you don't need to use `var` if you know the value during declaration, so you can safely remove this line, and line above can be rewritten as `searchTerm := "shadowed-private-key"`

Reply:
"Rookie" mistake. This PR was actually my first time programming in Go! 😆